docs/one-click-integrations/github-integration.md

```diff
@@ -112,6 +117,14 @@ The Pulse GitHub integration detects incidents automatically using [pull request
 -   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
 -   Pulse associates incidents to the system matching the repository name.
 
+### Use pull request naming convention (based on default branch) {: id="gh-incident-naming"}
+
+-   Pulse executes the same incident detection mechanism based on [pull request reverts](#gh-incident-pr-revert) using the naming convention you define instead of the default branch prefix `revert-`.
```

Comment:
I found this sentence a bit hard to read. Here's my attempt at simplifying it, let me know 😄 

```suggestion
-   Pulse can also flag incidents when it detects [pull request reverts](#gh-incident-pr-revert) using a branch prefix you define instead of the default branch prefix `revert-`.
```

Reply:
I think it's a bit strange that we mention branch prefix in this sentence but in the next sentence the prefix can be for the branch name or PR title.

Reply:
Fair point @afsmeira. @claudiacarpinteiro I'm updating my suggestion.

```suggestion
-   Pulse can also flag incidents when it detects [pull request reverts](#gh-incident-pr-revert) whose title or related branch name use a custom prefix.
```

Reply:
This is not exactly what I'm trying to say... I'll try to rephrase it.

@afsmeira For the custom incident detection, from where are we getting the number of the reverted pull request? Doe the rule for "Use pull request reverts" still apply?

Reply:
Yes, after we match the prefix we try to get the PR number that was reverted in the same way that we do for the non-custom detection.

Reply:
@nicklem @afsmeira I rephrased it, please have a look. Hope it's clear now. 🙏

Reply:
Looking good! 🚀